jr-objects/pom.xml

```diff
@@ -40,6 +40,11 @@ has no other dependencies, and provides additional builder-style content generat
       <artifactId>jackson-core</artifactId>
       <version>${jackson.version.core}</version>
     </dependency>
+    <dependency>
+      <groupId>it.unimi.dsi</groupId>
```

Comment:
What is this?

Reply:
It provides specialized Object2IntMap

Reverted here: 4e5851ca28a0f8c1f2d1171436ca0aa920ff0c7d

Reply:
Thanks! And yeah it's one of undocumented requirements (keeping deps minimal).
Thank you for your understanding.